db/migrate/20200331141420_regenerate_filters_taxonomy_searches.rb

```diff
@@ -0,0 +1,8 @@
+class RegenerateFiltersTaxonomySearches < ActiveRecord::Migration[5.2]
+  def up
+    Filter.where.not(:taxonomy_search => nil).find_each do |filter|
+      filter.valid?
+      filter.save! if filter.changed?
+    end
+  end
+end
```

Comment:
What is this for?

Reply:
This was a leftover from something I did some time ago and got accidentally commited